e2e-tests/ukef/cypress/support/trade-finance-manager-api/submitDeal.js

```diff
@@ -1,9 +1,8 @@
-const { submitDealAfterUkefIds, login } = require('./api');
+const { submitDealAfterUkefIds } = require('./api');
 const { T1_USER_1 } = require('../../../../e2e-fixtures/tfm-users.fixture');
 
 module.exports = (dealId, dealType) => {
   console.info('submitDeal::');
-  const { username, password } = T1_USER_1;
 
-  return login(username, password).then((token) => submitDealAfterUkefIds(dealId, dealType, null, token));
+  return cy.mockTfmLogin(T1_USER_1, null, false).then((token) => submitDealAfterUkefIds(dealId, dealType, null, token));
```

Comment:
I think the TFM login command should be object structured to avoid or make null/false values clearer

Otherwise here, it's not clear what null or false is for

Reply:
Yes, parameter are a bit messy at the moment, partly because it is used by TFM and UKEF tests, for browser and API calls.
Object structured params are fine.
Abhi is removing API calls in e2e-tests, so we will be able to simplify login command.

Reply:
Have just pushed this change, to be `tfmLogin` with object structured params